ee/clickhouse/system_status.py

```diff
@@ -111,12 +111,10 @@ def system_status() -> Generator[SystemStatusRow, None, None]:
     total_events_ingested_last_day = sync_execute(
         "SELECT count(*) as b from events WHERE _timestamp >= (NOW() - INTERVAL 1 DAY)"
     )[0][0]
-    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / total_events_ingested_last_day
+    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / total_events_ingested_last_day or 1
```

Comment:
this doesn't look right, see from python interpreter
```
>>> 50/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: division by zero
>>> 50/0 or 1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: division by zero
>>> 50/(0 or 1)
50.0
```

What we want here is likely:
```suggestion
    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / max(total_events_ingested_last_day, 1)
```